zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/application/AppConfigKey.kt

```diff
@@ -0,0 +1,7 @@
+package org.hexworks.zircon.api.application
+
+/**
+ * This simple interface is used to set and retrieve custom properties on [AppConfig]
+ * in a typesafe way.
```

Comment:
```suggestion
 * in a typesafe way.
 * 
 * @see AppConfigBuilder.withProperty
```

Reply:
Seems reasonable to me. Might as well mention AppConfig.getProperty too.